friture/audioproc.py

```diff
@@ -43,9 +43,6 @@ def __init__(self):
     def analyzelive(self, samples):
         samples = self.decimate(samples)
 
-        # uncomment the following to disable the decimation altogether
-        # decimation = 1
```

Comment:
I tried setting `self.decimation = 1` here, and that just resulted an error on the next line, "Operands cannot be broadcast together with shapes (4096,) (256,)"; so I think this comment is outdated

Reply:
(I also tried using `self.decimation = 1` and "fixing" the below line to make the shapes line up, but that just resulted in this error instead; I guess the value of self.decimation affects a bunch of places)
![image](https://user-images.githubusercontent.com/28263393/72958471-5d0fb580-3d75-11ea-9dc4-5a5403b67a94.png)

```
            fft = rfft(samples * repeat(self.window, len(samples)//len(self.window)))
```

Reply:
oh, lol, actually settings self.decimation = 1 works just fine, you just have to do it in the set_maxfreq function so that it never sets it to any non-1 value
![image](https://user-images.githubusercontent.com/28263393/73126380-e266b600-3f7f-11ea-8dba-919c36284ae8.png)